### PR TITLE
Optimizers in Machine Learning: #19

### PR DESCRIPTION
**Pull request by Anshika14528**
Explained about various optimization techniques which is used to change model weights and learning rates, like Gradient Descent, Stochastic Gradient Descent, Stochastic Gradient descent with momentum, Mini-Batch Gradient Descent, AdaGrad, RMSProp, AdaDelta, and Adam.

### CODE REVIEW

**Review by sanjay-kv — APPROVED** [review]

**Comment by sanjay-kv** [comment]
Hi, i wont be mergeing more PR on this repo, as this is only for beginner learning contribute and getting to PR. I believe I have merged around 7 Pr with level 1 , provide more merge will result leaderboard spamming.
